### DISCUSSION TITLE
cdn.tailwindcss.com should not be used in production.

### DISCUSSION THREAD

**Discussion by JS-Aibel** [zauberzeug/nicegui#2517] [Ideas / Feature Requests]
### Question

When creating a web application with NiceGUI a warning in the web browser’s Console is saying: 
`“cdn.tailwindcss.com should not be used in production.”`. 
How should we configure NiceGUI to solve this warning?

**Web browser Console warning:** 
_“cdn.tailwindcss.com should not be used in production. To use Tailwind CSS in production, install it as a PostCSS plugin or use the Tailwind CLI: https://tailwindcss.com/docs/installation”_
_“No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.”_

**Comment by falkoschindler** [comment]
Hi @JS-Aibel, see https://github.com/zauberzeug/nicegui/issues/337 for a related discussion.
At the moment we don't know how we can avoid this warning. Sure, this is not the intended way of using the Tailwind library. But since we can't run a Tailwind CLI in the NiceGUI context (or can we?), the current approach is a reasonable tradeoff.

**Reply by JS-Aibel** [reply]
Thank you for the reply, @falkoschindler.
I will continue the question in the other discussion.
